index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
+      </div>
+    </div>
+    <!-----------Header section------------->
+    <header>
+      <div class="main-header__brand">
+        <a href="index.html">
+          <img src="images/logo.png" alt="Logo" />
+        </a>
+      </div>
+    </header>
+
+    <section class="main-section">
+      <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
```

Comment:
- [ ] Kindly ensure that the color, font size, line-height, and font weight of the introductory text match the Figma design you chose. You can check out the Figma design specification for the exact properties used👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/191675844-e4157037-9a37-49da-aca4-0eac4db0b6b6.png)  | ![image](https://user-images.githubusercontent.com/63473584/191675970-2a980392-c268-4977-882c-ae5eb8e2ae1f.png)  |